test/test_sorted_containers.jl

```diff
@@ -1,18 +1,24 @@
-import Base.Ordering
-import Base.Forward
-import Base.Reverse
-import DataStructures.eq
+using Base: Ordering, Forward, Reverse, ForwardOrdering,
+            ReverseOrdering
+using DataStructures: IntSemiToken
 import Base.lt
-import Base.ForwardOrdering
-import Base.ReverseOrdering
-import DataStructures.IntSemiToken
+import DataStructures.eq
```

Comment:
We should make the imports into `using`s i think

Reply:
Done